optionals/compat_rhs_usf3/CfgWeapons.hpp

```diff
@@ -74,37 +74,70 @@ class CfgWeapons {
     };
     class rhs_weap_lmg_minimi_railed;
     class rhs_weap_m249_pip_S: rhs_weap_lmg_minimi_railed {
-        ACE_RailHeightAboveBore = 4.11044;
+        ACE_RailHeightAboveBore = 3;
         ACE_barrelLength = 348;
         ACE_barrelTwist = 177.8;
     };
     class rhs_weap_m249_pip_L: rhs_weap_lmg_minimi_railed {
-        ACE_RailHeightAboveBore = 4.34899;
+        ACE_RailHeightAboveBore = 3;
         ACE_barrelLength = 464.8;
         ACE_barrelTwist = 177.8;
     };
-    class rhs_weap_m240_base; // Rifle_Long_Base_F
-    class rhs_weap_m240B: rhs_weap_m240_base {
-        ACE_RailHeightAboveBore = 4.3987;
+    class rhs_weap_m249: rhs_weap_lmg_minimi_railed {
+        ACE_RailHeightAboveBore = 3;
+        ACE_barrelLength = 464.8;
+        ACE_barrelTwist = 177.8;
+    };
+    class rhs_weap_m249_pip: rhs_weap_lmg_minimi_railed {
+        ACE_RailHeightAboveBore = 3;
+        ACE_barrelLength = 464.8;
+        ACE_barrelTwist = 177.8;
+    };
+    class rhs_weap_M249_base;
+    class rhs_weap_m240_base: rhs_weap_M249_base {
+        ACE_RailHeightAboveBore = 2.68925;
         ACE_barrelTwist = 304.8;
         ACE_barrelLength = 629.92;
         ACE_Overheating_allowSwapBarrel = 1;
     };
-    class rhs_weap_m14ebrri: srifle_EBR_F {
+    class rhs_weap_m14_base: srifle_EBR_F {
         ACE_barrelTwist = 304.8;
         ACE_barrelLength = 558.8;
         ACE_Overheating_dispersion = 0.75;
-        ACE_RailHeightAboveBore = 3.08341;
+        ACE_RailHeightAboveBore = 1.98812;
     };
+    class rhs_weap_m14_socom_base: rhs_weap_m14_base {
+        ACE_barrelTwist = 304.8;
+        ACE_barrelLength = 413; //16.25 in (413 mm)
+        ACE_Overheating_dispersion = 0.75;
+        ACE_RailHeightAboveBore = 1.98812;
+    };
+    class rhs_weap_m14ebrri;
     class rhs_weap_sr25: rhs_weap_m14ebrri {
         ACE_barrelTwist = 285.75;
         ACE_barrelLength = 609.6;
-        ACE_RailHeightAboveBore = 3.13162;
+        ACE_RailHeightAboveBore = 1.98812;
     };
     class rhs_weap_sr25_ec: rhs_weap_sr25 {
         ACE_barrelTwist = 285.75;
         ACE_barrelLength = 508.0;
-        ACE_RailHeightAboveBore = 3.13689;
+        ACE_RailHeightAboveBore = 1.98812;
+    };
+    class rhs_weap_SCAR_H_Base;
+    class rhs_weap_SCAR_H_CQC_Base : rhs_weap_SCAR_H_Base {
+        ACE_barrelTwist =  304.8; // 1:12"
```

Comment:
Please only exactly one space before and after the equal sign.

Reply:
Changed all Entries to Space Equal Space ( = )